### PR TITLE
Add QQ group number.

### PR DESCRIPTION
**Pull request by chainsx**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -29,6 +29,19 @@
 |android|292 MB|17.7 MB|
 |ios|207 MB|3.97 MB|
 
+---
```
Comment (nihui) [inline README.md:32]:
```suggestion
```

File: README.md [hunk 2]
```diff
@@ -29,6 +29,19 @@
...
+<table>
+<tr>
+<td>
+<b>Technical Exchange QQ Group</b><br />
+623504742（opencv-mobile夸夸群）<br/>
+</td>
+</tr>
+</table>
+
+---
```
Comment (nihui) [inline README.md:43]:
```suggestion
```

**Comment by nihui** [comment]
Thanks for your contribution !

**Comment by SinyoWong** [comment]
请问qq群密码opencv-mobile全称是什么？

**Pull request merged**